ansible/ansible.cfg

```diff
@@ -28,3 +28,9 @@ ansible_python_interpreter = /opt/local/bin/python
 
 [hosts:freebsd]
 ansible_python_interpreter = /usr/local/bin/python
+
+[hosts:marist]
+ansible_become             = false
+ansible_python_interpreter = /NODEJS2/python-2017-04-12-py27/python27/bin/python
+home                       = /u
+server_jobs: 4
```

Comment:
This like is in the wrong format. I see it is in the host_vars template, was it forgotten here and should be removed?